Adding details at CFP page

railsstudent opened the pull request:
Related to #11
I created a second card to display details in Chinese.

railsstudent committed:
Adding details at CFP page

tonyhhyip commented on assets/pages/cfp/index.jinja:
```diff
@@ -49,16 +49,75 @@
+            </div>            
+        </div>
+        
+        <div class="card">
```
Add class `blue-theme` and `yellow-theme-text` for this card

tonyhhyip commented on assets/pages/cfp/index.jinja:
```diff
@@ -49,16 +49,75 @@
+                    <li>預計參加人數: 350 (與 2016 年出席人數相約)</li>
+                    <li>
+                        Link:
+                        <a href="https://goo.gl/forms/fmNjTshYMZlaaKo02" target="_blank">
```
Add class `link`

tonyhhyip commented on assets/pages/cfp/index.jinja:
```diff
@@ -49,16 +49,75 @@
+                    <li>Targeted number of participants: 350 (same as number of attendance in 2016)</li>
                     <li>
                         Link:
                         <a href="https://goo.gl/forms/fmNjTshYMZlaaKo02" target="_blank">
```
Add class link

tonyhhyip reviewed (changes requested):
Please add some class for styling

railsstudent committed:
Add details at CFP Page

1) Add link class to <li> element to show link
2) Add blue-theme and yellow-theme-text styling

railsstudent commented:
Done.
Add new style.scss class to add custom style

tonyhhyip commented on assets/pages/cfp/index.jinja:
```diff
@@ -44,11 +44,11 @@
 
-        <div class="card">
-            <div class="title">
+        <div class="card blue-theme">
```
Please refers to https://github.com/404busters/hkoscon-2017/blob/master/assets/pages/about/coc.jinja

tonyhhyip commented on assets/pages/cfp/index.jinja:
```diff
@@ -88,11 +88,11 @@
         
-        <div class="card">
-            <div class="title">
+        <div class="card blue-theme">
```
Please refers to https://github.com/404busters/hkoscon-2017/blob/master/assets/pages/about/coc.jinja

tonyhhyip commented on assets/scss/style.scss:
```diff
@@ -0,0 +1,13 @@
+
+.card {
```
Please modified in https://github.com/404busters/hkoscon-2017/blob/master/assets/scss/elements/_card.scss

tonyhhyip reviewed (changes requested):
Not expected result

railsstudent committed:
Modify scss style in _card.scss file and move yellow-theme-text to card level.

tonyhhyip reviewed (approved):
Alright, I will handle the rest
